SwiftWisdom/Core/Foundation/URL+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  URL+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Max Litteral on 7/17/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension URL: ExpressibleByStringLiteral {
+
+    public init(stringLiteral value: StaticString) {
+        self = URL(string: "\(value)")!
```

Comment:
I tried looking into that, but there doesn't seem to be a way using `ExpressibleByStringLiteral`. Could create an extension on String to add a `.url` property and have it optional.